### PR TITLE
MWPW-137345: [For Stage] Implementation of GenFill Interactive Marquee

### PR DESCRIPTION
**Pull request by drashti1712**
Implementation of GenFill Interactive Marquee

Resolves: [MWPW-137345](https://jira.corp.adobe.com/browse/MWPW-137345)

Test URLs:

Before: https://main--cc--adobecom.hlx.page/drafts/drashti/MWPW-137345/genfill-container?martech=off
After: https://stage-genfill--cc--adobecom.hlx.page/drafts/drashti/MWPW-137345/genfill-container?martech=off

For testing different viewports:

https://stage-genfill--cc--adobecom.hlx.page/drafts/drashti/MWPW-137345/genfill-container3?martech=off
https://stage-genfill--cc--adobecom.hlx.page/drafts/drashti/MWPW-137345/genfill-container2?martech=off

### CODE REVIEW

File: creativecloud/features/genfill/genfill-interactive.css [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  max-width: 30ch;
+}
+
+.enticement-text.light {
+color: #FFFFFF;
+}
+
+@media screen and (max-width: 600px) {
+  .genfill .mobile-media a,
+  .genfill .mobile-media picture {
```
Comment (aishwaryamathuria) [inline creativecloud/features/genfill/genfill-interactive.css:36]: if a is none we should not reset for picture

File: creativecloud/features/genfill/genfill-interactive.css [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  .genfill .tablet-media, 
+  .genfill .tablet-media a:first-child,
+  .genfill .tablet-media picture:first-child {
+    display: block;
+  }
+
+  /* Enticement */
+  .enticement-arrow {
+    width: 49px !important;
+    height: 73px !important;
```
Comment (aishwaryamathuria) [inline creativecloud/features/genfill/genfill-interactive.css:66]: Do not use important

File: creativecloud/features/genfill/genfill-interactive.css [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  .genfill .desktop-media, 
+  .genfill .desktop-media a:first-child, 
+  .genfill .desktop-media picture:first-child {
+    display: block;
+  }
+
+   /* Enticement */
+   .enticement-arrow {
+    width: 49px !important;
+    height: 73px !important;
```
Comment (aishwaryamathuria) [inline creativecloud/features/genfill/genfill-interactive.css:103]: Do not use important

File: creativecloud/features/genfill/genfill-interactive.css [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+    display: block;
+  }
+
+  .enticement-text {
+    position: absolute;
+    right: 345px;
+    top: -60px;
+    width: 240px;
+    display: block;
+    color: #000000;
```
Comment (aishwaryamathuria) [inline creativecloud/features/genfill/genfill-interactive.css:116]: Use milo text variables.

File: creativecloud/features/genfill/genfill-interactive.js [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+  clickConfig.autocycleInterval = setInterval(() => {
+    clickConfig.autocycleIndex = handleTransition(pics, clickConfig.autocycleIndex);
+    if (clickConfig.autocycleIndex === pics.length - 1) {
+      clearInterval(clickConfig.autocycleInterval);
+    }
+  }, interval);
+}
+
+function handleClick(aTags, clickConfig) {
+  aTags[0].style.display = 'block';
```
Comment (aishwaryamathuria) [inline creativecloud/features/genfill/genfill-interactive.js:26]: We already have 0th element set to block in CSS. We do not need to reset it again

**Review by salonijain3 — APPROVED** [review]

**Pull request merged**